templates/content/field--oe-document-reference--oe-title--oe-document-group.html.twig

```diff
@@ -0,0 +1,11 @@
+{#
+/**
+ * @file
+ * Field template of the document reference document group title entity.
```

Comment:
I see the comment was changed, but now it is very weird.
There is no "title entity".
Discussion in chat ongoing.